### PR TITLE
Add Docs

### PR DESCRIPTION
**Pull request by shadmansaleh**
Converted README.md to vim docs
with some help from https://github.com/FooSoft/md2vim

This should resolve issue #46

### CODE REVIEW

File: doc/lualine.txt [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+         ██▓     █    ██  ▄▄▄       ██▓     ██▓ ███▄    █ ▓█████
+        ▓██▒     ██  ▓██▒▒████▄    ▓██▒    ▓██▒ ██ ▀█   █ ▓█   ▀
+        ▒██░    ▓██  ▒██░▒██  ▀█▄  ▒██░    ▒██▒▓██  ▀█ ██▒▒███
+        ▒██░    ▓▓█  ░██░░██▄▄▄▄██ ▒██░    ░██░▓██▒  ▐▌██▒▒▓█  ▄
+        ░██████▒▒▒█████▓  ▓█   ▓██▒░██████▒░██░▒██░   ▓██░░▒████▒
+        ░ ▒░▓  ░░▒▓▒ ▒ ▒  ▒▒   ▓▒█░░ ▒░▓  ░░▓  ░ ▒░   ▒ ▒ ░░ ▒░ ░
+        ░ ░ ▒  ░░░▒░ ░ ░   ▒   ▒▒ ░░ ░ ▒  ░ ▒ ░░ ░░   ░ ▒░ ░ ░  ░
+          ░ ░    ░░░ ░ ░   ░   ▒     ░ ░    ▒ ░   ░   ░ ░    ░
+            ░  ░   ░           ░  ░    ░  ░ ░           ░    ░  ░
+
```
Comment (hoob3rt) [inline doc/lualine.txt:20]: I would remove this, looks wanky imo.

**Review by hoob3rt — COMMENTED** [review]
Yo! Sick find with md2vim, glad we don't have to rewrite the whole doc to vim

File: doc/lualine.txt (hunk 1 above)
Comment (shadmansaleh) [reply doc/lualine.txt:20]: What ... I thought it looked awsome.. Youre sure you wana remove it??

Comment (shadmansaleh) [reply doc/lualine.txt:20]: Are you looking at it from a full screen or on truncated github diff?

Comment (shadmansaleh) [reply doc/lualine.txt:20]: Maybe choose a different one from http://www.patorjk.com/software/taag/#p=testall&f=Avatar&t=lualine

**Comment by shadmansaleh** [comment]
>Yo! Sick find with md2vim, glad we don't have to rewrite the whole doc to vim

Yeah we should really be thanking FooSoft for writting it . Saw someone mention it in reddit in the morning :)

File: doc/lualine.txt (hunk 1 above)
Comment (hoob3rt) [reply doc/lualine.txt:20]: Full sreen obviously. Either way, it's not a big deal for me. Choose one which looks the best for you or drop it.

Comment (shadmansaleh) [reply doc/lualine.txt:18]: How about this one it's alot simpler

Comment (hoob3rt) [reply doc/lualine.txt:18]: Yep, I like it now. I will try to configure git hooks tomorrow and get it merged then

Comment (shadmansaleh) [reply doc/lualine.txt:18]: > I will try to configure git hooks tomorrow and get it merged then

What git hooks ?

Comment (hoob3rt) [reply doc/lualine.txt:18]: It would be nice to have the doc automatically update whenever we update README.md, will save some pain.

Comment (shadmansaleh) [reply doc/lualine.txt:18]: It wonn't work as it's not output from md2vim . There were problems in the output. I've used it's output as a templet to create this . So it's a modified version of it . Trying to automate doc creation on edit to README.md will be even more pain.

Comment (hoob3rt) [reply doc/lualine.txt:18]: Ok, thanks for letting me know. It's all good then

**Pull request merged**